### DISCUSSION TITLE
Repeater Not working for Second Table

### DISCUSSION THREAD

**Discussion by burhanahmed92** [filamentphp/filament#2763] [Help]
I have two Models (Order & OrderDetail). I created an Order Resource and was able to save data in Orders Table very easily using Form Element. But at the same time, I want to add data in another table too. Another Table is order_detail. 
So First I Fill out the information for the Order and then i use repeater to add information in Order Details

```
Repeater::make('order_detail')
    ->schema([
        TextInput::make('order_id')->required(),
        TextInput::make('product_name')->required(),
        TextInput::make('quantity')->required(),
    ])
    ->columns(2)
```

But I am unable to do that. It gives JSON serializable error. I want to save the order first and then return the order id which i use in the repeater to complete the one-to-many table relationship.
Any idea how can i do it using Single Resource.

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/forms/fields#populating-automatically-from-a-hasmany-relationship
